eoscostarica/eos-rate
# refactor(smart-contract): update contract to eden clsdk interpretation

### Steps to test

1. Check [this tutorial](https://github.com/eoscommunity/demo-clsdk) base configuration
1. mkdir build && cd build
1. cmake `clsdk-cmake-args` ..
1. make -j
1. Check compilation is right

Final result could be checked at [this account](https://jungle3.bloks.io/account/smarttestacc)
Note: `smarttestacc` is a personal test account, in future, may be its `abi` and `wasm` files may have been changed.
